### ISSUE TITLE
Discuss length of the foss-responders initiative

### ISSUE THREAD

**Issue by nuritzi** [foss-responders/about#8]
FOSS-responders is a pop-up response to the COVID-19 pandemic, to help FOSS organizations that are dealing with the ramifications of canceling events. 

Since people and orgs are donating money, I think it'd be good to define at least an initial strategy for the length of this initiative. For example, when will we start phasing it out and how will we deal with any left over funds? Do we see this group and community staying together somehow or merging with another community? 

Having some idea of a duration timeline for this will also help the initial participants of this group set expectations around their availability, and help find reinforcements / build a team if necessary. 

## Guiding Questions
 * Should we set an initial duration timeline and then reassess? What are the factors that determine the need for our response? 
 * What do we do with any extra funds, if we get to that? 
 * What do we do with the community that we build?

**Comment by DuaneOBrien** [comment]
Capturing some relevant things that were discussed in other channels previously:

**How long do we keep funds** We briefly discussed the idea if setting a trigger to redistribute the funds based on the most recent request. By way of example, if we got no requests for aid after 6 months, we would redistribute the funds so they could be used elsewhere;

**What do we do with any extra funds?** We briefly discussed two options - donating them back to open source collective so they could redeploy them, or donating them to travel scholarship programs as determined by the committee who reviews financial aid applications;

For both of these questions, no decision was reached. There seemed to be general consensus that when we land on a decision for these points, we need to document it publicly so donors know what the plan is.

**Comment by DuaneOBrien** [comment]
Re: What do we do with this community - I don't think we've spent any time discussing "What happens after this has been resolved" but I think it's a good question to keep in mind. I could see one of two options: look for another similar problem to solve, or document and suspend operations in a way that would enable us to restart should the working group be needed again. 

My instinct says that when we've served our purpose, we should suspend operations until there's another large scale crisis to which we need to respond. If we go that route, we should make it clear how we will know it's time, and make it clear how the decision is reached.
